code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -120,6 +136,49 @@ public void updateAccount(Account account, String name, String kind, String exte
         }
     }
 
+    @Override
+    public List<AccountRecord> getAccessibleProjects (Long userId, List<String> orgIds, List<String> teamIds) {
```

Comment:
Please return List<? extends Account>.  Record objects shouldn't be referenced outside the DAO layer.